### ISSUE TITLE
LibreOffice Calc: NVDA have no access to Organize Template menu

### ISSUE THREAD

**Issue by surfer0627** [nvaccess/nvda#6908]
The bug prevents NVDA from accessing the menu options in the Organize Template menu of LibreOffice Calc. This menu allows users to perform actions such as renaming, deleting, and applying templates as default.

### Steps to reproduce:

1. Open LibreOffice Calc
2. Change the column width by navigating to Format > Column > Width.
3. Save the changes by pressing Shift+F11, entering a name for the new template, choosing "My Templates" under Templates - Categories, and clicking Ok.
4. Press ctrl+shift+n to Organize template.
5. Press tab to move to categories combo box "My Templates"
6. Press tab to move to list.
7. Select your template and press applications key to open menu.

#### Actual behavior:

NVDA fails to access the menu, preventing the user from accessing options such as renaming, deleting, and applying as the default template.

#### Expected behavior:

NVDA have access to the template menu.

#### NVDA installed/portable/running from source:
Portable
#### NVDA version:
2023.2rc1
#### Windows version:
Windows version 10.0.19042
#### Name and version of other software in use when reproducing the issue:
LibreOffice version 7.5.4.2

**Comment by Adriani90** [comment]
This is still reproducible in libre office calc 6.2.2 and NVDA 2019.1. I guess this is an external fix. @feerrenrut, @leonardder what do you think?

**Comment by Adriani90** [comment]
Here is the developer info after pressing the context menu key on the template to open its menu:

```
Developer info for navigator object:
name: u'Vorlagen verwalten'
role: ROLE_DIALOG
states: STATE_FOCUSABLE, STATE_MODAL
isFocusable: True
hasFocus: False
Python object: <NVDAObjects.Dynamic_DialogIAccessibleWindowNVDAObject object at 0x053BCD50>
Python class mro: (<class 'NVDAObjects.Dynamic_DialogIAccessibleWindowNVDAObject'>, <class 'NVDAObjects.behaviors.Dialog'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: ''
location: RectLTWH(left=1184, top=506, width=832, height=688)
value: None
appModule: <'soffice' (appName u'soffice', process ID 12716) at address 53bbdb0>
appModule.productName: u'LibreOffice'
appModule.productVersion: u'6.2.2.2'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 3933772L
windowClassName: u'SALSUBFRAME'
windowControlID: 0
windowStyle: -1798569984
windowThreadID: 5316
windowText: u'Vorlagen verwalten'
displayText: u't'
IAccessibleObject: <POINTER(IAccessible2) ptr=0x123662d4 at 4f76d50>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=3933772, objectID=-4, childID=0
IAccessible accName: u'Vorlagen verwalten'
IAccessible accRole: ROLE_SYSTEM_DIALOG
IAccessible accState: STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_SIZEABLE, STATE_SYSTEM_VALID (1179648)
IAccessible accDescription: u''
IAccessible accValue: None
IAccessible2 windowHandle: 3933772
IAccessible2 uniqueID: -1023
IAccessible2 role: ROLE_SYSTEM_DIALOG
IAccessible2 states: IA2_STATE_MODAL (256)
IAccessible2 attributes: exception: (-2147467259, 'Unbekannter Fehler', (None, None, None, 0, None))
```

**Comment by Adriani90** [comment]
cc: @michaelweghorn not sure if there is already a bug filled with LO on this?

**Comment by surfer0627** [comment]
a bug reported to Libreoffice:

[157001 – screen reader NVDA Access Issue in LibreOffice Calc's Organize Template Menu](https://bugs.documentfoundation.org/show_bug.cgi?id=157001)

**Comment by michaelweghorn** [comment]
This is fixed in LibreOffice (25.8) now, see https://bugs.documentfoundation.org/show_bug.cgi?id=157001

**Comment by Adriani90** [comment]
Thank you for the update, closing as works for me.
